text/text.md

```diff
@@ -72,50 +72,57 @@ OK, jungle. And you tell tall tales around the fire, right?
 OK, you all know that, no hooting, on with the story. But you all
-reckon now I'm a loyal Republic citizen, ain't I? 
+reckon now I'm a loyal Republic citizen, right? 
 
 So. It was early winter, but the weather was still sunny and mild. I was walking
-along the docks, as I was wont to do. The air was clear and I could see ships
-docking and leaving, ships from all our sister nations of the world, Spain, Cuba, 
-Colombia, Puerto Rico, but also from not-so-friendly places 
-who decided the buck needn't stop here and had resumed trade as soon as the
+along the docks, as I like to do. The air was clear, and I could see ships
+docking and leaving, ships from all our sister nations of the world, like Spain, Cuba, 
+Colombia, and Puerto Rico. There were also some from not-so-friendly places 
+who decided the buck #Not sure what the buck is? needn't stop here and had resumed trade as soon as the
 smoke from the Freedom War settled down and become the smoke of the
-smokestacks built during the Planazo. Formerly unfriendly places like  
-the imperialist United Kingdom, the Austrohungarian Empire, those bohunks... The
-air swarmed also with the airships about to dock or leave from the
-airship port, the Spanish clover-fronted ones, the elegant Cubans,
-even the odd bulky and slumbering American, you could just spend the
+smokestacks built during the Planazo. 
+####This is a run-on sentence, but I'm not sure what the best way to divide this is. 
+Formerly unfriendly places like the imperialist United Kingdom, the Austrohungarian Empire, those bohunks... 
+####What is a bohunk?
+The air swarmed also with the airships about to dock or leave from the
+airship port.  There were the Spanish clover-fronted ones, the elegant Cubans, and
+even the odd bulky and slumbering American. You could just spend the
 whole day looking at them, slowly realizing they were no longer in the
 place you had first spotted them, in the middle of a cloud, or watching them
 appearing nose first from the middle of another. Yes, Gummy, they do look like that 
 sometimes. If that's what you're thinking about. Which you, no doubt, are.
 
 I was there, looking up and down while strolling alongside the docks, minding my 
 own business, which was precisely what I was looking for at that early moment of 
-the day. I needed a hire, was looking to work for somebody who needed a quick 
+the day. 
+#####A bit awkward here, but the idea is clear.  The phrasing just needs a little work. 
+I was looking to work for somebody who needed a quick 
 mind and a strong pair of arms, but meanwhile I kept dodging the sun, keeping 
-myself in the shade, because the sun made my clothes too warm for comfort. 
+myself in the shade, because the sun made my clothes too warm for comfort.
+#####Maybe to make the transition between who the narrator is addressing you can use markers, like stars or hashtags or spaces? 
```

Comment:
In principle, he's addressing the whole hobo jungle. Those below are actually month names.